gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -46,11 +46,32 @@
 
 #include <string.h>
 #include "tensor_converter.h"
-#include "converter-media-info.h"
+#ifndef NO_VIDEO
```

Comment:
IMO, this change will break build, if disable-video-support=true or disable-audio-support=true.

Reply:
Thanks for pointing it out. Refactored headers to support them.